lms/static/sass/course/layout/_courseware_header.scss

```diff
@@ -41,12 +42,16 @@
         display: block;
         text-align: center;
         text-decoration: none;
+        box-sizing: border-box;
+        border-bottom: 1px solid transparent;
```

Comment:
I would prefer this to be the same 4px as in the hover/focus/active case. I think you could make that work by shifting the padding bottom on this element too, which would be better than the current hard-coded 13px.